### PR TITLE
Fix typos, comments, unused imports

### PR DESCRIPTION
**Pull request by megaserg**
## Type of changes

- [ ] Bug fix
- [ ] New feature
- [x] Documentation / docstrings
- [ ] Tests
- [ ] Other

## Checklist

- [x] I've run the latest [black](https://github.com/psf/black) with default args on new code.
- [ ] I've updated CHANGELOG.md and CONTRIBUTORS.md where appropriate.
- [ ] I've added tests for new code.
- [x] I accept that @willmcgugan may be pedantic in the code review.

## Description

Just some semi-automatically found typos.

### CODE REVIEW

File: rich/style.py [hunk 1]
```diff
@@ -345,7 +345,7 @@ def link(self) -> Optional[str]:
         return self._link
 
     @property
-    def transaprent_background(self) -> bool:
+    def transparent_background(self) -> bool:
```
Comment (willmcgugan) [inline rich/style.py:348]: This one is technically a breaking change. Would you mind documenting this in  CHANGELOG.md

**Review by willmcgugan — COMMENTED** [review]
Looks good. If you wouldn't mind documenting the silly typo with background_style in the changelog.

File: rich/style.py (hunk 1 above)
Comment (ssbarnea) [reply rich/style.py:348]: Indeed breaking but I doubt anyone used it and failed to make a PR or at least raise a bug. Still, updating changelog seems reasonable.

**Review by ssbarnea — APPROVED** [review]

File: rich/style.py (hunk 1 above)
Comment (megaserg) [reply rich/style.py:348]: Done! FWIW, here's a full GitHub search for that token: https://github.com/search?q=transaprent_background&type=code 😅

**Comment by willmcgugan** [comment]
Thanks @megaserg I'll take that as a lesson to be careful about cutting and pasting typos!

**Pull request merged**